src/components/login/Login.tsx

```diff
@@ -1,15 +1,47 @@
+import { FormEvent, useState } from 'react'
+
+import { useNavigate } from 'react-router-dom'
+
+import LoginCredentials from '../../interfaces/LoginCredentials'
+import { getLogin } from '../../services/auth.service'
+import { setToken } from '../../services/token.service'
+
 import './Login.css'
 
-function Login() {
+function Login(props: { setIsLogged : React.Dispatch<React.SetStateAction<boolean>> }) {
+  const [ user, setUser ] = useState({ email: '', password: '' })
+  const [ errorMessage, setErrorMessage ] = useState ('')
+  const navigate = useNavigate()
+
+  const handleSubmit = (e : FormEvent) => {
+    e.preventDefault()
+    const request: LoginCredentials = {
+      email: user.email,
+      password: user.password
+    }
+    getLogin(request).then(response => {
+      setToken(response.data.accessToken)
+      props.setIsLogged(true)
+      navigate('/')
+    }).catch(() => {setErrorMessage('Wrong email and/or password.')})
```

Comment:
It would be nice to check error status code - for example you can get 5xx status for server problems and still you will get 'Wrong email and/or password.' message which may not be the case